earth_enterprise/src/installer/uninstall_fusion.sh

```diff
@@ -159,21 +160,23 @@ determine_os()
     local test_versionid=""
 
     if [ -f "$OS_RELEASE1" ] || [ -f "$OS_RELEASE2" ]; then
-		if [ -f "$OS_RELEASE1" ]; then
-	        test_os="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^NAME= | sed 's:name=::gI')"
-    	    test_versionid="$(cat $OS_RELEASE1 | sed -e 's:\"::g' | grep ^VERSION_ID= | sed 's:version_id=::gI')"
-		else
-			test_os="$(cat $OS_RELEASE2 | sed 's:[0-9\.] *::g')"
-			test_versionid="$(cat $OS_RELEASE2 | sed 's:[^0-9\.]*::g')"
-		fi
+	if [ -f "$OS_RELEASE1" ]; then
```

Comment:
Hey Chris -- it does look a little crazy, right?  The issue here is that I wrote the scripts in one editor and then all of my recent work has been done in nano (for the most part).  The tabbing has been inconsistent between the two.  I'll need to mod that process.  :)